include/z64save.h

```diff
@@ -1141,7 +1141,7 @@ typedef enum {
 #define WEEKEVENTREG_63_20 PACK_WEEKEVENTREG_FLAG(63, 0x20)
 #define WEEKEVENTREG_63_40 PACK_WEEKEVENTREG_FLAG(63, 0x40)
 
-// showed Couple's Mask to meeting
+// Showed Couple's Mask to meeting
 #define WEEKEVENTREG_63_80 PACK_WEEKEVENTREG_FLAG(63, 0x80)
```

Comment:
I guess we could also name this one. Maybe `WEEKEVENTREG_SHOWED_COUPLES_MASK_TO_MAYOR_DOTOUR_MEETING`. Kinda too verbose tho